src/containers/ClusterTriggerBinding/ClusterTriggerBinding.test.js

```diff
@@ -70,76 +67,17 @@ it('ClusterTriggerBindingContainer renders', async () => {
       clusterTriggerBindingName
     }
   };
-  const middleware = [thunk];
-  const mockStore = configureStore(middleware);
-  const testStore = mockStore({
-    clusterTriggerBindings: {
-      byId: {},
-      errorMessage: null,
-      isFetching: false
-    }
-  });
 
   const { getByText } = renderWithRouter(
-    <Provider store={testStore}>
-      <ClusterTriggerBindingContainer
-        intl={intl}
-        match={match}
-        fetchClusterTriggerBinding={() => Promise.resolve()}
-        error={null}
-        loading={false}
-      />
-    </Provider>
+    <ClusterTriggerBindingContainer
+      intl={intl}
+      match={match}
+      fetchClusterTriggerBinding={() => Promise.resolve()}
+      error={null}
+      loading={false}
```

Comment:
Leave out loading and error ?

edit don't need to make that change, see it was preexisting in a few places. Up to you

Reply:
I didn't want to make too many changes here to keep the diff relatively simply, and we'll need to revisit some of these tests anyway with the next upcoming change.